doc/sg/c_cpp_coding_style.md

```diff
@@ -86,6 +86,27 @@ void sort(int array[], size_t size){
 }
 ```
 
+This also applies to `struct` and `enum` members:
+
+```c
+/**
+ * Boot data stored in the flash info partition.
+ */
+typedef struct boot_data {
+  /**
+   * CRC32 checksum of boot data.
+   *
+   * The region covered by this checksum starts immediately after this field and
+   * ends at the end of the entry.
+   */
+  uint32_t checksum;
+  /**
+   * Boot data identifier.
+   */
+  uint32_t identifier;
```

Comment:
Thanks for including fields for both long and short docs!

nit: I know this is just an example but we just updated the definition of this type:

```suggestion
  /**
   * SHA-256 digest of boot data.
   *
   * The region covered by this digest starts immediately after this field and
   * ends at the end of the entry.
   */
  hmac_digest_t digest;
  ...
  /**
   * Boot data identifier.
   */
  uint32_t identifier;
  ...
```

Reply:
Good spot, thanks!